presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConfig.java

```diff
@@ -153,4 +160,95 @@ public KafkaConfig setMessagesPerSplit(int messagesPerSplit)
         this.messagesPerSplit = messagesPerSplit;
         return this;
     }
+
+    @Config("kafka.security.protocol")
```

Comment:
Can we move these configuration details to a separate class like `KafkaSecurityConfig` ?

Reply:
will do

Reply:
Can we move all these config getters and setter fo the `KafkaSecurityConfig` ?

Reply:
Would that work without using a separate Config file?